scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -378,12 +394,71 @@ https://${docsDomain}/components/lanes`;
   }
 }
 
+export class LaneReadmeRemoveCmd implements Command {
```

Comment:
I would make the class name the same as the command name
the command name is remove readme, so better to call the class LaneRemoveReadme
(same for the add readme command)